Core.cs

```diff
@@ -12,10 +13,12 @@ public class Core
 
         internal static Settings ModSettings = new Settings();
         internal static string ModDirectory;
-
+        internal static string KnockdownPhrasePath;
+        
         public static void Init(string directory, string settingsJSON)
         {
             ModDirectory = directory;
+            KnockdownPhrasePath = Path.Combine(ModDirectory, "phrases.txt");
```

Comment:
You should move this to after the settings are loaded, and into a helper function inside the `FallingHandling` class.